src/cryptenroll/cryptenroll.h

```diff
@@ -13,6 +13,12 @@ typedef enum EnrollType {
         _ENROLL_TYPE_INVALID = -EINVAL,
 } EnrollType;
 
+typedef enum UnlockType {
+        UNLOCK_PASSWORD,
+        UNLOCK_KEYFILE,
+        UNLOCK_FIDO2,
```

Comment:
_MAX and _INVALID

Reply:
Do we need _INVALID here though? Because `arg_unlock_type` defaults to `UNLOCK_PASSWORD` anyway, and _INVALID will never be used.